charts/incubator/shlink/Chart.yaml

```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: "v2.10.1"
+appVersion: "2.10.1"
```

Comment:
as I explained: This gets overwritten by the CI.
It does not mater what you enter here, it gets deleted.